pyproject.toml

```diff
@@ -15,6 +15,7 @@ dependencies = [
   'nestd==0.3.1',
 # conditional
   'uvloop == 0.17.0; sys_platform == "darwin"',
+  'uvloop == 0.17.0; sys_platform == "linux"',
```

Comment:
Can we do `sys_platform != "win32"`?
Example [here](https://packaging.python.org/en/latest/specifications/core-metadata/#requires-external-multiple-use)

Reply:
@AntoineRR , just `win32` will not work. Windows has arm support now and even `x86_64` possibly. Not sure how windows architecture is mentioned

Reply:
I think you are confusing several things, `arm` and `x86_64` are CPU architectures, while windows is the OS. `sys_platform` calls `sys.platform` which gets specification of the OS : https://docs.python.org/3/library/sys.html#sys.platform

Reply:
> while windows is the OS. sys_platform calls sys.platform which gets specification of the OS :

Oh, thanks @AntoineRR . TIL 😄 

I will update the PR